pkg/ddl/tests/partition/db_partition_test.go

```diff
@@ -3673,7 +3676,7 @@ func TestPrimaryNoGlobalIndex(t *testing.T) {
 	// get the full row with a single primary key lookup.
 	// Since a global index would first need to be checked to get the partition to
 	// do a second index lookup for getting the full row.
-	tk.MustContainErrMsg(`create table t (a int primary key clustered, b varchar(255)) partition by key(b) partitions 3`, `A CLUSTERED INDEX must include all columns in the table's partitioning function`)
+	tk.MustContainErrMsg(`create table t (a int primary key clustered, b varchar(255)) partition by key(b) partitions 3`, `A PRIMARY KEY must include all columns in the table's partitioning function`)
```

Comment:
Why this changed? Use CLUSTERED INDEX is more precise.

Reply:
Fixed it :) Thanks for noticing!